Mi 27. Sep 11:39:45 CEST 2023
htc-cmp146
/home/htc/bzfkostr/GitCodePSO/Unstable-PSO/Experiments/1D

Hello! This is task number 98
Mi 27. Sep 11:39:58 CEST 2023
Mi 27. Sep 11:39:58 CEST 2023
htc-cmp146
/home/htc/bzfkostr/GitCodePSO/Unstable-PSO/Experiments/1D
Traceback (most recent call last):
  File "/home/htc/bzfkostr/GitCodePSO/Unstable-PSO/Experiments/1D/Measures1D.py", line 20, in <module>
    DivergentG=np.load(path+'G_s'+str(nameDivergent[function_number])+str(s)+'.npy')
  File "/usr/lib/python3/dist-packages/numpy/lib/npyio.py", line 416, in load
    fid = stack.enter_context(open(os_fspath(file), "rb"))
FileNotFoundError: [Errno 2] No such file or directory: '/home/htc/bzfkostr/SCRATCH/SimulationsPSO/Data/G_s0DivergentMu0.14035087719298245Particle10T200012.npy'
6
Traceback (most recent call last):
  File "/home/htc/bzfkostr/GitCodePSO/Unstable-PSO/Experiments/1D/Exploration1D.py", line 88, in <module>
    ExplorationDivergent=fct_local_minima('Divergent', LocalMinimaSchwefel,T_PSO)
  File "/home/htc/bzfkostr/GitCodePSO/Unstable-PSO/Experiments/1D/Exploration1D.py", line 56, in fct_local_minima
    X=np.load(path+'X_s'+str(name)+str(s)+'.npy')
  File "/usr/lib/python3/dist-packages/numpy/lib/npyio.py", line 416, in load
    fid = stack.enter_context(open(os_fspath(file), "rb"))
FileNotFoundError: [Errno 2] No such file or directory: '/home/htc/bzfkostr/SCRATCH/SimulationsPSO/Data/X_s6DivergentMu0.14035087719298245Particle10T200012.npy'
Mi 27. Sep 11:40:01 CEST 2023
